libcxx/test/libcxx/depr/depr.default.allocator/allocator_types.cxx2a.pass.cpp

```diff
@@ -1,51 +0,0 @@
-//===----------------------------------------------------------------------===//
```

Comment:
I have the same question here, I don't understand why this is removed in all standard modes.

Reply:
Restored the file, changed it to run in pre-C++20 modes (and all the other deleted files too)